internal/support/scheduler/router.go

```diff
@@ -101,12 +101,7 @@ func loadRestRoutes(r *mux.Router, dic *di.Container) {
 	interval.HandleFunc(
 		"/{"+ID+"}",
 		func(w http.ResponseWriter, r *http.Request) {
-			restDeleteIntervalByID(
-				w,
-				r,
-				bootstrapContainer.LoggingClientFrom(dic.Get),
-				container.DBClientFrom(dic.Get),
-				schedulerContainer.QueueFrom(dic.Get))
+			restDeleteIntervalByID(w, r, bootstrapContainer.LoggingClientFrom(dic.Get), container.DBClientFrom(dic.Get), schedulerContainer.QueueFrom(dic.Get), container.DBClientFrom(dic.Get), container.DBClientFrom(dic.Get))
```

Comment:
Can we put these arguments on new lines like it was previously?

Reply:
Can we look into applying this previous comment?

Reply:
I missed this comment, and the analogous one below and will address the both this minute ✅